src/test/regress/sql/rpt.sql

```diff
@@ -601,6 +601,22 @@ drop table if exists t1;
 drop table if exists t2;
 drop function if exists f(i int);
 
+-- start_ignore
+drop table if exists tbl2;
+drop table if exists tbl1;
+-- end_ignore
+
+create table tbl2 (a numeric, b varchar(255), c numeric) distributed replicated;
+create table tbl1 (a bigserial, b varchar(15), c varchar(15)) distributed replicated;
+
+explain (analyze off, costs off, verbose off)
```

Comment:
what does this test check? write the comments. And in patch description your are talking about some problem in a really abstract way, add at least 1 sentence specifying the problem.

Reply:
done